examples/fullstack/frontend/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: 8264cb3e8a797eef39cbcd32bb56fd07790efb7f
+  channel: dev
```

Comment:
Is there anything dev specific in this app, or should it work with flutter stable channel?

Reply:
Yes ... `clearSnackBars` requires dev channel. I'd like to leave it, but commented it out for now.